src/components/elements/Map-mapbox/MapControls/ViewImageCarousel.tsx

```diff
@@ -48,16 +45,19 @@ const ViewImageCarousel = ({ modelFilesData }: { modelFilesData: GetV2MODELUUIDF
     ];
   }, [modelFilesData]);
 
-  const openFormModalHandlerImageGallery = () => {
-    openModal(<ModalImageGallery onClose={closeModal} tabItems={modelFilesTabItems} title={""} />);
-  };
-
+  const [openModal, setOpenModal] = useState(false);
   return (
     <div className="relative">
-      <Button variant="white-button-map" className="flex items-center gap-2" onClick={openFormModalHandlerImageGallery}>
+      <Button variant="white-button-map" className="flex items-center gap-2" onClick={() => setOpenModal(!openModal)}>
         <Icon name={IconNames.IMAGE_ICON} className="h-4 w-4" />
         <Text variant="text-12-bold"> {t("View Gallery")}</Text>
       </Button>
+      <ModalImageGallery
+        onClose={() => setOpenModal(false)}
+        tabItems={modelFilesTabItems}
+        title={""}
+        WrapperClassName={`${!openModal ? "hidden" : ""}`}
```

Comment:
There doesn't seem to be any reason to wrap this in a template string; can be this instead:
```
WrapperClassName={openModal ? "hidden" : ""}
```

Reply:
done